docs/testrunner-toolkit/configuration/common-syntax.md

```diff
@@ -297,7 +300,76 @@ __Example__:
   mode: "sauce"
 ```
 
-> For all available resolutions please visit [this documentation page](https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options#TestConfigurationOptions-SauceLabsCustomTestingOptions).
+## `artifacts`
+
+__Description__: Allows you to define configuration relative to artifacts.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+artifacts:
+  download:
+    when: always
+    match:
+      - junit.xml
+    directory: ./artifacts/
+```
+
+### `download`
+
+__Description__: Allows you to define configuration relative to artifacts downloads.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  download:
+    when: always
+    match:
+      - junit.xml
+    directory: ./artifacts/
+```
+
+#### `when`
+
+__Description__: Defines when to download the artifacts.
+
+__Type__: *string*
+
+__Values__:
+- `always`: Artfiacts will always be downloaded.
+- `never`: Artifacts will never be downloaded.
+- `pass`: Artifacts will be downloaded only if suite pass.
+- `fail`: Artifacts will be downloaded only if suite fails.
+
+__Example__:
+```yaml
+    when: always
+```
+
+#### `match`
+
+__Description__: Defines the list of name patterns which artifacts must match to be downloaded. Wildcard character (`*`) can be used.
+
+__Type__: *string[]*
+
+__Example__:
+```yaml
+    - junit.xml
+    - *.log
+```
+
+#### `directory`
+
+__Description__: Defines the folder where artifacts will be downloaded. Each suite will have its own subdirectory.
```

Comment:
```suggestion
__Description__: Specifies the path to the folder location in which artifacts will be downloaded. Each suite will have its own subdirectory.
```

@FriggaHel -- how does this work? this value is a parent folder and saucectl will create child folders for each suite in the test project?

Reply:
That's exactly this !